angr/analyses/reassembler.py

```diff
@@ -1196,7 +1196,7 @@ def shrink(self, new_size):
                 # it's not aligned?
                 raise BinaryError('Fails at Data.shrink()')
 
-            pointers = self.size / pointer_size
+            pointers = int( self.size / pointer_size )
```

Comment:
`pointers = self.size // pointer_size` is better,

Reply:
Oh 
Does `pointer_size` always supposed to be 1 according to the logic?
In my experiment, sometimes `pointer_size` is 4 when I prints logs.

Reply:
`pointer_size` should be either 4 (for 32-bit programs) or 8 (for 64-bit programs).